resources/tests.mjs

```diff
@@ -446,29 +414,26 @@ Suites.push({
     },
     tests: [
         new BenchmarkTestStep('Adding' + numberOfItemsToAdd + 'Items', page => {
-            const newTodo = page.querySelector(".new-todo");
+            const newTodo = page.querySelector('.new-todo');
             for (let i = 0; i < numberOfItemsToAdd; i++) {
-                newTodo.value = 'Something to do ' + i;
-                newTodo.dispatchEvent(new Event('input', {
-                  bubbles: true,
-                  cancelable: true
-                }));
-                processElmWorkQueue(page);
-                page.triggerEnter(newTodo, 'keydown');
-                processElmWorkQueue(page);
+                newTodo.setValue('Something to do ' + i);
+                newTodo.dispatchEvent('input');
+                page.eval('processElmWorkQueue()');
```

Comment:
Nit: it feels like these kinds of things should be handled by test-specific hooks, rather than just opening up the doors for evals in tests? I don't expect this specific change to change the performance characteristics in Firefox as the code directly inside the eval isn't hot and isn't doing anything weird, but it still makes me a little nervous. I would prefer something like `page.call('processElmWorkQueue')`.